tests/integration/conftest.py

```diff
@@ -36,6 +39,25 @@
 }
 
 
+@pytest.fixture(scope="function", name="http")
+def fixture_http_client():
+    """Return the --test-flask-image test parameter."""
+    retry_strategy = Retry(
+        total=5,
+        connect=5,
+        read=5,
+        other=5,
+        backoff_factor=5,
+        status_forcelist=[429, 500, 502, 503, 504],
+        allowed_methods=["HEAD", "POST", "GET", "OPTIONS"],
+        raise_on_status=False,
+    )
+    adapter = HTTPAdapter(max_retries=retry_strategy)
+    http = requests.Session()
+    http.mount("http://", adapter)
+    return http
```

Comment:
How about using:
'''suggestion
with requests.Session() as session:
    ...
    yield session
'''
So that we close the connection safely when we're done using it :)